assets/constants.ts

```diff
@@ -4906,10 +4910,18 @@ export default {
     "code": "es",
     "description": "español (Spanish)",
     "direction": "ltr"
+  }, {
+    "code": "sw",
+    "description": "kiswahili (Swahili)",
+    "direction": "ltr"
   }, {
     "code": "sv",
     "description": "svenska (Swedish)",
     "direction": "ltr"
+  }, {
+    "code": "ta",
```

Comment:
We need to run job to generate opportunties for these new lanaguges, right?

Reply:
Notes from offline discussion with @DubeySandeep: when new language codes are added to this list,  `ExplorationOpportunitySummaryModelRegenerationJob` job needs to be run. This is handled in the release process by a script, but it does not work for cherry-picked PRs.